plugins/@grouparoo/mongo/src/lib/table-import/getSampleRows.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  GetSampleRowsMethod,
+  DataResponseRow,
+} from "@grouparoo/app-templates/dist/source/table";
+import { castRow } from "./util";
+
+export const getSampleRows: GetSampleRowsMethod = async ({
+  connection,
+  appOptions,
+  sourceOptions,
+  tableName,
+}) => {
+  const out: DataResponseRow[] = [];
+  if (sourceOptions) {
+    const fields = sourceOptions.fields.toString().split(",");
```

Comment:
`trim` here too. maybe `DRY` that up.

Reply:
There were spots before when you were replacing the tbale name and such. is that always the case?